packages/kit/src/exports/vite/index.js

```diff
@@ -289,16 +287,12 @@ function kit({ svelte_config }) {
 					__SVELTEKIT_DEV__: 'true',
 					__SVELTEKIT_EMBEDDED__: kit.embedded ? 'true' : 'false'
 				};
-
-				new_config.ssr = {
-					// Without this, Vite will treat `@sveltejs/kit` as noExternal if it's
-					// a linked dependency, and that causes modules to be imported twice
-					// under different IDs, which breaks a bunch of stuff
-					// https://github.com/vitejs/vite/pull/9296
-					external: ['@sveltejs/kit', 'cookie', 'set-cookie-parser']
```

Comment:
any idea why `cookie` and `set-cookie-parser` were in here?

Reply:
hmm. you added it in https://github.com/sveltejs/kit/pull/8429, but I can't see why it'd be needed

Reply:
That's strange, because my guess would be that we don't need `external` anymore _because_ of #8429 (since things are treated as external and noExternal now anyway, which is why we need that hack of patching things up)

Reply:
The SvelteKit tests use pnpm which links SvelteKit. To make sure it also works in an unlinked scenario, I directly modified `node_modules/@sveltejs/kit/src/exports/vite/index.js` in one of my projects and `build` successfully completed there as well. I believe this is unnecessary and can be removed. (If that turns out to be wrong for any reason, I'll definitely add a comment explaining in the revert PR :wink:)

Reply:
It wouldn't affect the `build` output because it's not in the `if (is_build)` block. `pnpm dev` in a project that links `@sveltejs/kit` fails with this change